templates/registration/logged_out.html

```diff
@@ -0,0 +1,6 @@
+{% extends "base.html" %}
+
+{% block content %}
+  <p>Logged out!</p>
+  <a href="{% url 'login'%}">Click here to login again.</a>
```

Comment:
There should be a space between the 'login' and the closing '%}' in the URL tag. It should be `{% url 'login' %}`.